Trivago/Jade/Application/JsonApi/Request/RequestFactory.php

```diff
@@ -283,6 +298,20 @@ private function parseFilters(ResourceConfig $resourceConfig, CompositeFilter $c
         }
     }
 
+    /**
+     * Validating the "path" is not under "excluded_attributes" config section
+     * @param ResourceConfig $resourceConfig
+     * @param $key
+     * @param $path
+     */
+    private function validateExcludedPaths(ResourceConfig $resourceConfig, $key, $path)
+    {
+        if ($this->strictFilteringAndSorting && $resourceConfig->isAttributeExcluded($resourceConfig->getRealPath($path))) {
+            throw InvalidRequest::createWithMessage($key, 'invalid_path', sprintf('The path "%s" is a non rendered column. 
```

Comment:
One last concern: Since this is a security matter maybe it's not a good idea to expose the fact that the column exists.
Maybe use the debug flag and if it's true show this message. If not then just act as the column does not exists.